src/app/app.routing.ts

```diff
@@ -0,0 +1,28 @@
+import { Routes } from "@angular/router";
+import { DogComponent } from "./dog/dog.component";
+import { DogsComponent } from "./dogs/dogs.component";
+import { NotFoundComponent } from "./not-found/not-found.component";
+import { SubBreedComponent } from "./sub-breed/sub-breed.component";
+
+export const AppRoutes: Routes = [
+    {
+        path: '',
+        component: DogsComponent
+    },
+    {
+        path: 'breed/:name',
+        component: DogComponent
```

Comment:
in order to avoid the confusion between DogComponent and DogsComponent you can rename the DogComponent In BreedComponent